From 76429cc01cf60a04c3dd28ca32543c0763282f0b Mon Sep 17 00:00:00 2001
From: "H. Vetinari" <h.vetinari@gmx.com>
Date: Sat, 2 Nov 2024 15:41:34 +1100
Subject: [PATCH 1/4] disable gcsfs_test

it cannot work unless we package https://github.com/googleapis/storage-testbench,
which however has extremely tight dependencies on protobuf etc., making it very
hard to fit this into our migration patterns
---
 cpp/src/arrow/filesystem/CMakeLists.txt | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/cpp/src/arrow/filesystem/CMakeLists.txt b/cpp/src/arrow/filesystem/CMakeLists.txt
index 7afdf566f..b22a48a0c 100644
--- a/cpp/src/arrow/filesystem/CMakeLists.txt
+++ b/cpp/src/arrow/filesystem/CMakeLists.txt
@@ -42,14 +42,6 @@ if(ARROW_BUILD_BENCHMARKS)
                       ${ARROW_BENCHMARK_LINK_LIBS})
 endif()
 
-if(ARROW_GCS)
-  add_arrow_test(gcsfs_test
-                 EXTRA_LABELS
-                 filesystem
-                 EXTRA_LINK_LIBS
-                 google-cloud-cpp::storage)
-endif()
-
 if(ARROW_AZURE)
   add_arrow_test(azurefs_test
                  EXTRA_LABELS
